ConvertNumberToWords/Program.cs

```diff
@@ -11,48 +11,120 @@ class Program
         static void Main()
         {
             // TODO : keiskite FROM..TO skaicius pagal tai kiek spesite padaryt uzduociu. (-19...19, -99..99, ir tt.)
-            const int FROM_NUMBER = -9;
-            const int TO_NUMBER = 9;
+            const int FROM_NUMBER = -999;
+            const int TO_NUMBER = 999;
 
             Console.WriteLine("Sveiki!");
 
             string numberInput = "";
 
-            while(numberInput != " ")
+            while (numberInput != " ")
             {
                 Console.WriteLine($"Iveskite skaiciu nuo {FROM_NUMBER} iki {TO_NUMBER}.   (Iveskite Space programai uzbaigti.)");
                 numberInput = Console.ReadLine();
-
-
-                // TODO :au jusu programa!
+                int num = Convert.ToInt32(numberInput);
```

Comment:
Sita veiksma reikia daryti po to kai isitikinama ar skaicius teisingas.. kitaip programa nuluz ivedus neteinsingai.